pkgs/applications/office/portfolio/default.nix

```diff
@@ -25,15 +26,21 @@ let
 
   runtimeLibs = lib.makeLibraryPath [ gtk3 webkitgtk ];
 in
-stdenv.mkDerivation rec {
+maven.buildMavenPackage rec {
   pname = "PortfolioPerformance";
   version = "0.67.0";
 
-  src = fetchurl {
-    url = "https://github.com/buchen/portfolio/releases/download/${version}/PortfolioPerformance-${version}-linux.gtk.x86_64.tar.gz";
-    hash = "sha256-bm38t8wgLEYZVDsMnOv3Wj6TgKu2s36wo8q0Sj7pPNI=";
+  src = fetchFromGitHub {
+    owner = "buchen";
+    repo = "portfolio";
+    rev = version;
+    hash = "sha256-bZCWjJ0+yDu/oGGutebh5ZSaCF3lyn/ghSo73mpM8Po=";
   };
 
+  mvnHash = "sha256-gbb/lhobNlAHggWIiAuPbQMYBak3b8F0mvLNEJ7sVOk=";
```

Comment:
hash missmatch

Reply:
I've executed the build twice and obtained two distinct hashes for the Maven dependencies. If I recall correctly, I previously attempted to build portfolio from its source some time ago but abandoned the endeavor when I encountered difficulty obtaining a consistent hash.

Reply:
Indeed, sad. I'm going to close the PR, it was an opportunistic PR and I do not have the motivation to deal with some Maven shenanigan